packages/oui-clipboard/clipboard.less

```diff
@@ -0,0 +1,52 @@
+@import './_variables';
+
+.oui-clipboard {
```

Comment:
This should be a modifier of `.oui-input-group` (eg. `oui-input-group_clipboard`).
We should reuse our existing styles if possible to avoid duplicate :)